google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/scrape/metadata_scraper.py

```diff
@@ -151,3 +165,25 @@ def _get_topic_compaction_config(self, config):
             })
 
         return topic_compaction_config
+
+    def _get_topic_schemas(self, topic_name):
+        topic_schemas = {}
+        subject_value = topic_name + '-value'
+        subject_key = topic_name + '-key'
+        try:
+            subjects = self._schema_registry_client.get_subjects()
+            if subject_value in subjects:
+                value_schema = self._schema_registry_client.get_latest_version(
```

Comment:
Do you have metadata about the schema version? If you do, should this be ingested?

Reply:
Yes, it is easy to get metadata about the schema version. I think it may be useful? It shows the evolution of the schema. I'll scrape it too then

Reply:
done